client/me/purchases/cancel-private-registration/index.jsx

```diff
@@ -30,14 +31,22 @@ const CancelPrivateRegistration = React.createClass( {
 		page( paths.managePurchase( domain, id ) );
 	},
 
-	cancel() {
+	cancel( event ) {
+		event.preventDefault();
+		event.target.blur();
```

Comment:
We could add a comment explaining why we're calling `blur()` - it's not obvious at first glance.

Just curious, why did you have to call `preventDefault()`?